add doc for v5.1: Deploy -> Install TiKV (#231)
tikv/website: content/docs/5.1/deploy/install/verify.md

---
title: Verify Cluster Status
description: Check the cluster status and connect to the cluster
menu:
    "5.1":
        parent: Install TiKV
        weight: 3
---

> After a TiKV cluster is deployed, you need to check whether the cluster runs normally. This document introduces how to check the cluster status using TiUP commands and Grafana, and how to connect to the TiKV cluster to perform simple `put` and `get` operations.

Reviewer: ```suggestion
After a TiKV cluster is deployed, you need to check whether the cluster is up and running. This document will introduce the methods to check the cluster status using TiUP and Grafana, and will also describe how to connect to the TiKV cluster using a TiKV client to do simple `put` and `get` operations.
```

## Check the TiKV cluster status

This section describes how to check the TiKV cluster status using TiUP commands and Grafana.

### Use TiUP

Use the `tiup cluster display <cluster-name>` command to check the cluster status. For example:

```bash
tiup cluster display tikv-test
```

Expected output: If the `Status` information of each node is `Up`, the cluster runs normally.

### Use Grafana

1. Log in to the Grafana monitoring at `${Grafana-ip}:3000`. The default username and password are both `admin`.

2. To check the TiKV port status and load monitoring information, click **Overview**.

> ## Connect to TiKV and perform simple operations

Reviewer: ```suggestion
## Connect to TiKV and do simple operations
```

> This section describes how to connect to the TiKV cluster to perform simple `put` and `get` operations.

Reviewer: ```suggestion
This section describes how to connect to the TiKV cluster to do simple `put` and `get` operations.
```
